lib/ports.json

```diff
@@ -13030,6 +13030,23 @@
     ],
     "code": "53398"
   },
+  "INKAK": {
+    "name": "Kakinada",
+    "city": "Kakinada",
+    "province": "Andhra Pradesh",
+    "country": "India",
+    "alias": [],
+    "regions": [],
+    "coordinates": [
+      82.2475,
+      16.9891
+    ],
+    "timezone": "Asia/Calcutta",
+    "unlocs": [
+      "INAGR"
```

Comment:
@Jean-MarcCheynier did you mean `INKAK` instead of `INAGR` ?

`INAGR` is Agra unloc

Reply:
https://www.cbp.gov/sites/default/files/assets/documents/2017-Feb/appendix_f_0.pdf

It's the CBP port code, `53398` is actually the correct one, some port share the same code.